README.md

```diff
@@ -1,92 +1,199 @@
+```bash
+python -m daal4py my_application.py
+```
+Patches can also be enabled programmatically:
+```python
+from sklearn.svm import SVC
+from sklearn.datasets import load_digits
+
+svm_sklearn = SVC(kernel="rbf", gamma="scale", C=0.5)
+
+start = time()
+svm_sklearn = svm_sklearn.fit(X, y)
+end = time()
+print(start-end) # output: 0.141261...
+print(svm_sklearn.score(X, y)) # output: 0.9905397885364496
 
-## Examples
+from daal4py.sklearn import patch_sklearn
+patch_sklearn() # <-- apply patch
+from sklearn.svm import SVC
 
-Examples for different programming languages:
+svm_d4p = SVC(kernel="rbf", gamma="scale", C=0.5)
 
-- [C++](https://github.com/oneapi-src/oneDAL/tree/master/examples/cpp)
-- [C++ SYCL*](https://github.com/oneapi-src/oneDAL/tree/master/examples/cpp_sycl)
-- [Java](https://github.com/oneapi-src/oneDAL/tree/master/examples/java)
+start = time()
+svm_d4p = svm_d4p.fit(X, y)
+end = time()
+print(start-end) # output: 0.032536...
+print(svm_d4p.score(X, y)) # output: 0.9905397885364496
+```
+
+
+For more details browse our [daal4py documentation](https://intelpython.github.io/daal4py/).
+
+
+## DAAL Samples
+
+<img align="right" style="display:inline;" height=300 width=550 src="https://github.com/PivovarA/daal/raw/dev/apivovar-newReadme/.github/charts/intel%20DAAL%20Spark%20samples%20vs%20Apache%20Spark%20MLlib.png"></a>
+
+Samples is a non official part of a project, that contains examples of how DAAL can be used in different applications.
+
+**DAAL Spark Samples is a good example.**
+DAAL provides scala / java interfaces that match Apache Spark MlLib API and use DAAL solvers under the hood. This implementation allows you to get a 3-18X increase in performance compared to default Apache Spark MLlib.
+
+  ## Examples
+
+Except C++ and Python API DAAL also provide API for C++ SYCL and Java languages. Check out tabs below for more examples.
+- [C++](https://github.com/intel/daal/tree/master/examples/daal/cpp)
+- [C++ SYCL*](https://github.com/intel/daal/tree/master/examples/daal/cpp_sycl)
+- [Java](https://github.com/intel/daal/tree/master/examples/daal/java)
 
 Data Examples for different computation modes:
 
-- [Batch](https://github.com/oneapi-src/oneDAL/tree/master/examples/data/batch)
-- [Distributed](https://github.com/oneapi-src/oneDAL/tree/master/examples/data/distributed)
-- [Online](https://github.com/oneapi-src/oneDAL/tree/master/examples/data/online)
+- [Batch](https://github.com/intel/daal/tree/master/examples/daal/data/batch)
+- [Distributed](https://github.com/intel/daal/tree/master/examples/daal/data/distributed)
+- [Online](https://github.com/intel/daal/tree/master/examples/daal/data/online)
 
 ## Documentation
-
-- [Get Started](http://oneapi-src.github.io/oneDAL/getstarted.html)
-- [System Requirements](https://software.intel.com/en-us/articles/system-requirements-for-oneapi-data-analytics-library#)
```

Comment:
why did you remove this?

updated link to system requirements:
```
- [System Requirements](https://software.intel.com/content/www/us/en/develop/articles/system-requirements-for-oneapi-data-analytics-library.html)
```

Reply:
Thanks. I updated this item